pkg/upgrade/upgrade.go

```diff
@@ -140,13 +139,21 @@ func HasDeleteConfigMap(c client.Client) bool {
 // createDefaultDSC creates a default instance of DSC.
 // Note: When the platform is not Managed, and a DSC instance already exists, the function doesn't re-create/update the resource.
 func CreateDefaultDSC(cli client.Client, platform deploy.Platform) error {
+	// Set the default DSC name depending on the platform
+	var DSCName string
+	if platform == deploy.ManagedRhods || platform == deploy.SelfManagedRhods {
+		DSCName = "rhods"
```

Comment:
sorry, now I understand what you said in the meeting. I don't think this should be named "rhods". Can we keep it as "default" as well? Otherwise, we might want to use a name that is more resilient to change, like "default-dsc" or similar.